Formula/ninja.rb

```diff
@@ -31,6 +30,9 @@ def install
     bin.install "ninja"
     bash_completion.install "misc/bash-completion" => "ninja-completion.sh"
     zsh_completion.install "misc/zsh-completion" => "_ninja"
+    doc.install "doc/manual.asciidoc"
+    elisp.install "misc/ninja-mode.el"
+    (share/"vim/vimfiles/syntax").install "misc/ninja.vim"
```

Comment:
These things seemed useful to install, but probably not worth a `revision` bump.